doc/news/new_opmath.rst

```diff
@@ -166,28 +163,18 @@ Summary of the update
 
     **qml.Hamiltonian**
 
-    The legacy classes :class:`~.pennylane.operation.Tensor` and :class:`~.pennylane.Hamiltonian` will soon be deprecated.
-    :class:`~.ops.op_math.LinearCombination` offers the same API as :class:`~.pennylane.Hamiltonian` but works well with new opmath classes.
+    The classes :class:`~.pennylane.operation.Tensor` and :class:`~.pennylane.Hamiltonian` have been removed.
+    :class:`~.ops.op_math.LinearCombination` offers the same API as :class:`~.pennylane.Hamiltonian` but works well
+    with new opmath classes.

```

Comment:
```suggestion
    The classes :class:`~.pennylane.operation.Tensor` and :class:`~.pennylane.ops.Hamiltonian` have been removed. The familiar ``qml.Hamiltonian`` can still be used, which dispatches to ``LinearCombination`` and offers the same usage and functionality but
      with different implementation details.
```

Then we can remove the sentence below (won't let me suggest on all four of these lines at once 😿 )